.github/workflows/ci.yml

```diff
@@ -50,6 +50,12 @@ jobs:
       - name: Cargo sort
         run: make cargo-sort
 
+      - name: Cargo Machete
+        uses: bnjbvr/cargo-machete@main
+        with:
+          args: --with-metadata
+        continue-on-error: false
```

Comment:
@Xuanwo Should we fail the ci or remove the dependencies using `--fix`?

Reply:
> remove the dependencies using

Hi, please fail the ci as we can contributors to remove the unused deps.

Reply:
Please remove this if we using default value.